pkg/termios/termios_plan9.go

```diff
@@ -44,45 +54,63 @@ func NewWithDev(device string) (*TTYIO, error) {
 
 // NewTTYS returns a new TTYIO.
 func NewTTYS(port string) (*TTYIO, error) {
-	f, err := os.OpenFile(filepath.Join("/dev", port), os.O_RDWR, 0o620)
+	f, err := os.OpenFile(filepath.Join("/dev", port), os.O_RDWR, 0)
 	if err != nil {
 		return nil, err
 	}
 	return &TTYIO{f: f}, nil
 }
 
 // GetTermios returns a filled-in Termios, from an fd.
+// And, sorry, on Plan 9, there seems to be no way to
+// find out if it is in hold/raw mode. Odd.
+// Because the ctl file is separate,
 func GetTermios(fd uintptr) (*Termios, error) {
-	return &Termios{}, nil
+	f, err := consctlFile("/", fd)
+	if err != nil {
+		return nil, err
+	}
+	return &Termios{mode: reset, ctl: f}, nil
 }
 
-// Get terms a Termios from a TTYIO.
+// Get a Termios from a TTYIO.
 func (t *TTYIO) Get() (*Termios, error) {
 	return GetTermios(t.f.Fd())
 }
 
 // SetTermios sets tty parameters for an fd from a Termios.
-func SetTermios(fd uintptr, ti *Termios) error {
-	return fmt.Errorf("Plan 9: not yet")
+// The only thing we can do is write to mode to the ctl
+func SetTermios(_ uintptr, t *Termios) error {
+	if t.ctl == nil {
+		return fmt.Errorf("Termios ctl is not set up:%w", os.ErrInvalid)
```

Comment:
we have a linter, but it only works on linux/amd64 as I see, but errors string should not be capitalised.